daemon/api.go

```diff
@@ -956,6 +965,119 @@ func deleteCapability(c *Command, r *http.Request) Response {
 	}
 }
 
+// SkillGrant is a part of response from getSkills().
```

Comment:
In fact, these should probably be private as well. No good reason to expose such implementation details from the daemon.